plugins/system/debug/src/DataCollector/ProfileCollector.php

```diff
@@ -82,8 +93,8 @@ public function __construct(Registry $params)
 	 * @param   string|null  $label      Public name
 	 * @param   string|null  $collector  The source of the collector
 	 *
-	 * @since  4.0.0
 	 * @return void
+	 * @since  4.0.0
```

Comment:
https://developer.joomla.org/coding-standards/docblocks.html

```
@return (required, followed by a blank line)
```

```suggestion
	 * @return void
	 *
	 * @since  4.0.0
```